elasticsearch/probe/readiness.sh

```diff
@@ -43,21 +41,3 @@ function check_if_ready() {
 }
 
 check_if_ready "/" "Elasticsearch node is not ready to accept HTTP requests yet"
-check_if_ready "/.searchguard.$DC_NAME" "Searchguard index '.searchguard.$DC_NAME' is missing in ES cluster"
```

Comment:
we're removing this because we're going to move to single SG index pattern?

Reply:
We are removing it because it was never propagated to the ES5 branch.  We decided to significantly simply the readiness probe some time ago because all the checks we devised interfered with pod going into a readiness state.  I saw this behavior when testing the images produced downstream by @adammhaile but I don't know why we don't see issues in upstream CI.  It could be a timing issue